### PR TITLE
Create MSTest

### PR DESCRIPTION
**Pull request by tgspn**

### CODE REVIEW

File: src/Grok.Net.sln [hunk 1]
```diff
@@ -1,9 +1,11 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.28922.388
+# Visual Studio 15
```
Comment (Marusyk) [inline src/Grok.Net.sln]: please don't commit this change

Comment (tgspn) [reply src/Grok.Net.sln]: fix

**Comment by Marusyk** [comment]
@tgspn are you going to add more tests?

**Comment by tgspn** [comment]
Yep, I will implement in incremental steps

**Comment by Marusyk** [comment]
Hi @tgspn, any updates?

**Pull request closed without merging**